edb/pgsql/metaschema.py

```diff
@@ -4131,121 +4131,128 @@ async def bootstrap(
-        dbops.CreateFunction(NormalizeArraySliceIndexFunction()),
-        dbops.CreateFunction(IntOrNullFunction()),
-        dbops.CreateFunction(ArrayIndexWithBoundsFunction()),
-        dbops.CreateFunction(ArraySliceFunction()),
-        dbops.CreateFunction(StringIndexWithBoundsFunction()),
-        dbops.CreateFunction(LengthStringProxyFunction()),
-        dbops.CreateFunction(LengthBytesProxyFunction()),
-        dbops.CreateFunction(SubstrProxyFunction()),
-        dbops.CreateFunction(StringSliceImplFunction()),
-        dbops.CreateFunction(StringSliceFunction()),
-        dbops.CreateFunction(BytesSliceFunction()),
-        dbops.CreateFunction(JSONIndexByTextFunction()),
-        dbops.CreateFunction(JSONIndexByIntFunction()),
-        dbops.CreateFunction(JSONSliceFunction()),
-        dbops.CreateFunction(DatetimeInFunction()),
-        dbops.CreateFunction(DurationInFunction()),
-        dbops.CreateFunction(DateDurationInFunction()),
-        dbops.CreateFunction(LocalDatetimeInFunction()),
-        dbops.CreateFunction(LocalDateInFunction()),
-        dbops.CreateFunction(LocalTimeInFunction()),
-        dbops.CreateFunction(ToTimestampTZCheck()),
-        dbops.CreateFunction(ToDatetimeFunction()),
-        dbops.CreateFunction(ToLocalDatetimeFunction()),
-        dbops.CreateFunction(StrToBool()),
-        dbops.CreateFunction(BytesIndexWithBoundsFunction()),
-        dbops.CreateEnum(SysConfigSourceType()),
-        dbops.CreateEnum(SysConfigScopeType()),
-        dbops.CreateCompositeType(SysConfigValueType()),
-        dbops.CreateFunction(ConvertPostgresConfigUnitsFunction()),
-        dbops.CreateFunction(InterpretConfigValueToJsonFunction()),
-        dbops.CreateFunction(PostgresConfigValueToJsonFunction()),
-        dbops.CreateFunction(SysConfigFullFunction()),
-        dbops.CreateFunction(SysConfigFunction()),
-        dbops.CreateFunction(ResetSessionConfigFunction()),
-        dbops.CreateFunction(ApplySessionConfigFunction(config_spec)),
-        dbops.CreateFunction(SysGetTransactionIsolation()),
-        dbops.CreateFunction(GetCachedReflection()),
-        dbops.CreateFunction(GetBaseScalarTypeMap()),
-        dbops.CreateFunction(GetTypeToRangeNameMap()),
-        dbops.CreateFunction(GetPgTypeForEdgeDBTypeFunction()),
-        dbops.CreateFunction(DescribeInstanceConfigAsDDLFunctionForwardDecl()),
-        dbops.CreateFunction(DescribeDatabaseConfigAsDDLFunctionForwardDecl()),
-        dbops.CreateFunction(DescribeRolesAsDDLFunctionForwardDecl()),
-        dbops.CreateRange(Float32Range()),
-        dbops.CreateRange(Float64Range()),
-        dbops.CreateRange(DatetimeRange()),
-        dbops.CreateRange(LocalDatetimeRange()),
-        dbops.CreateFunction(RangeToJsonFunction()),
-        dbops.CreateFunction(RangeValidateFunction()),
-        dbops.CreateFunction(RangeUnpackLowerValidateFunction()),
-        dbops.CreateFunction(RangeUnpackUpperValidateFunction()),
-    ])
+    commands.add_commands(
```

Comment:
I don't think the expanded indentation here makes things better and it churns up the diff

Reply:
No, it doesn't make it better, but this is what black does.

I'm using "darker", that formats only code chunks that I've changed and that's why this was formatted.

I had reverted it a few times, but I'd be very happy if we could just merge the change, so I don't have to keep reverting it all the time I work on metaschema.

Reply:
Sigh. Maybe do something like
```
new = [
   ...
]
commands.add_commands(new)
```